Add new MBL message for #4183 and translation

3d-gussner opened the pull request:
Please review the translations

- [x] cs :czech_republic:  @DRracer 
- [x] de :de: @3d-gussner @sarusani
- [x] es :es: 
- [X] _fr :fr:_  Exceeds limits and will not be updated
- [ ] hr :croatia:  @Prime1910
- [x] hu :hungary: @AttilaSVK @Hauzman
- [x] it :it: @wavexx 
- [x] nl :netherlands: @vintagepc @stelgenhof
- [x] no :norway: @OS-kar @trondkla
- [x] pl :poland: Feedback from @arekm 
- [x] ro :romania: @leptun @Hauzman
- [x] sk :slovakia: @ingbrzy @shatter136
- [x] sv :sweden:  @Painkiller56

3d-gussner committed:
Add new MBL message for #4183 and translate these messages

ingbrzy commented on lang/po/Firmware_sk.po:
```diff
@@ -2555,6 +2555,11 @@ msgstr "Výmena filamentu M600. Vložte nový filament alebo vysuňte starý."
+#. MSG_MBL_FAILED_Z_CAL c=20 r=4
+#: ../../Firmware/Marlin_main.cpp:2976
+msgid "Mesh bed leveling failed. Please run Z calibration."
+msgstr "Vyrovnanie lôžka zlyhalo. Spustite kalibráciu Z."
```
Should be
"Vyrovnanie platne zlyhalo. Spustite kalibráciu Z."

leptun commented on lang/po/Firmware_ro.po:
```diff
@@ -2573,6 +2573,11 @@ msgstr ""
+#. MSG_MBL_FAILED_Z_CAL c=20 r=4
+#: ../../Firmware/Marlin_main.cpp:2976
+msgid "Mesh bed leveling failed. Please run Z calibration."
+msgstr "Nivelarea patului a esuat. Nivelarea patului a esuat."
```
Translation is not correct. I propose this, though I'm open to suggestions since it still sounds somewhat weird.
`Nivelarea patului a eșuat. Rulează Calibrare Z.`

Ro3Deee replied:
the proposed text by @leptun  is ok. It sounds  somewhat weird because there are no established words for this actions in romanian (maybe „Execută calibrarea [axei] Z” is more appropriate for „Rulează Calibrează Z”).

leptun replied:
I tried to word it in such a way that the "Calibrare Z" in the string here is the same as the menu item in the Calibrations menu. The grammar is a bit wonky because of that, but I'm not sure how to word it better. If we could use quotations around "Calbrare Z", it would sound a lot better in my opinion, but I don't think the translation scripts handle this.

Hauzman replied:
I agree with @leptun I did not find better translation

AttilaSVK commented:
Hi! I adjusted the Hungarian so the printer is addressing the user
consistently (you as in Du instead of Sie in German, Ty instead of Vy in
Czech/Slovak) and added diacritics to the first part of the string as
well: Sikertelen asztal szintezés. Kérjük, futtasd a Z kalibrálást

3d-gussner ***@***.***> ezt írta (időpont: 2023. szept. 13.,
Sze, 11:22):

> Please review the translations
>
>    - cs 🇨🇿 @DRracer <https://github.com/DRracer>
>    - de 🇩🇪 @3d-gussner <https://github.com/3d-gussner>
>    - es 🇪🇸
>    - fr 🇫🇷
>    - hr 🇭🇷 @Prime1910 <https://github.com/Prime1910>
>    - hu 🇭🇺 @AttilaSVK <https://github.com/AttilaSVK> @Hauzman
>    <https://github.com/Hauzman>
>    - it 🇮🇹 @wavexx <https://github.com/wavexx>
>    - nl 🇳🇱 @vintagepc <https://github.com/vintagepc> @stelgenhof
>    <https://github.com/stelgenhof>
>    - no 🇳🇴 @OS-kar <https://github.com/OS-kar> @trondkla
>    <https://github.com/trondkla>
>    - pl 🇵🇱
>    - ro 🇷🇴 @leptun <https://github.com/leptun> @Hauzman
>    <https://github.com/Hauzman>
>    - sk 🇸🇰 @ingbrzy <https://github.com/ingbrzy> @shatter136
>    <https://github.com/shatter136>
>    - sv 🇸🇪 @Painkiller56 <https://github.com/Painkiller56>
>
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/prusa3d/Prusa-Firmware/pull/4386
> Commit Summary
>
>    - ca27841
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/commits/ca27841d6778771c6a746b3c2bd2fa1880665950>
>    Add new MBL message for #4183 and translate these messages
>
> File Changes
>
> (15 files <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files>)
>
>    - *M* Firmware/Marlin_main.cpp
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-a2e2d6e9dab1e37ea9a2882433e2b8f70af751b3c3445d5e997d11a93d4b28b3>
>    (2)
>    - *M* lang/po/Firmware.pot
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-14fedb8dfdce511a0bdcf251bbe0fc1c53d69fcec094186519cc6f57826b4b1b>
>    (5)
>    - *M* lang/po/Firmware_cs.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-0aa4aafa0e88956cdefd904d3a4d90d8075a6baea4b7a7552fa25663b8550f7c>
>    (5)
>    - *M* lang/po/Firmware_de.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-7aa3bbee27c68325a9362a2508a9dbdffb40452a4296772d7b6075b7cafd7d43>
>    (5)
>    - *M* lang/po/Firmware_es.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-94fdeb509cca7d35e68df03af06931c6c67805acb0c427946922c1b5caa91549>
>    (5)
>    - *M* lang/po/Firmware_fr.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-e1c029266b2a50654a6638406d3cfe838eeb4502bde0aadca6cdcb4372d42a44>
>    (5)
>    - *M* lang/po/Firmware_hr.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-487d5469566575627c7dc47abf437c52e890de941c1fd8d4db5a2a4d6ef08d64>
>    (5)
>    - *M* lang/po/Firmware_hu.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-3a7d51d07fb16c60d273b7986aa88a7892939b19a9bad4f3501a153ffb0604e9>
>    (5)
>    - *M* lang/po/Firmware_it.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-d948679300ec3cbaa7e705150dfcaee8932cf6f047e800ca1036ff551f31e81c>
>    (5)
>    - *M* lang/po/Firmware_nl.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-92de134d7cfe75b62e86a4e83716cd95550b11f2ddee228150a9c6faa4d6a34b>
>    (5)
>    - *M* lang/po/Firmware_no.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-22e5dd5d54111e837f3b40db65f2a126b09cc4157dcfa1f34b6e5dde0bf4a607>
>    (5)
>    - *M* lang/po/Firmware_pl.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-67132bccb4c23fbeaf41753633298c3047956124c6a42a0a8a574e3c042a09e0>
>    (5)
>    - *M* lang/po/Firmware_ro.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-6774f424618c20ccb1d99bf93cb34bab0b6ee03a55868fa7021a11ef6c28667f>
>    (5)
>    - *M* lang/po/Firmware_sk.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-9584ba5c6a2b7618c16dc96fa21d692e9d5d69c393c55f37166bb05e98557fbc>
>    (5)
>    - *M* lang/po/Firmware_sv.po
>    <https://github.com/prusa3d/Prusa-Firmware/pull/4386/files#diff-cf46655e0192f28112da2985d28dc728949010730a0ac629dbb7049f9f53a357>
>    (5)
>
> Patch Links:
>
>    - https://github.com/prusa3d/Prusa-Firmware/pull/4386.patch
>    - https://github.com/prusa3d/Prusa-Firmware/pull/4386.diff
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/prusa3d/Prusa-Firmware/pull/4386>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIZ6Q5MVJ7LXNQMD2VLODNTX2F3OVANCNFSM6AAAAAA4WDKE5E>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

sarusani commented:
@3d-gussner
Shouldn't it be "MeshBett Ausgleich fehlgeschlagen. Z Kalibrierung ausführen." instead of "MeshBett Ausgleich fehlgeschlagen. Führe Z Kalibrierung aus."?

3d-gussner commented:
Thanks for the fast responses, i will wait a bit to collect all changes in one or two commits.

Painkiller56 commented:
Swedish translation good.

stelgenhof commented:
Dutch translation looks good to me.

Commit:
Update CZ translation

wavexx commented:
Italian also looks good to me.

OS-kar commented on lang/po/Firmware_no.po:
```diff
@@ -2549,6 +2549,11 @@ msgstr "M600 filamentskifte. Sett inn en ny filament eller løs ut den gamle."
+#. MSG_MBL_FAILED_Z_CAL c=20 r=4
+#: ../../Firmware/Marlin_main.cpp:2976
+msgid "Mesh bed leveling failed. Please run Z calibration."
+msgstr "Sengeplanering feilet. Kjør Z-kalibrering."
```
```suggestion
msgstr "Plateplanering feilet. Kjør Z-kalibrering."
```

trondkla replied:
Not sure about this correction. Sengeplanering is used elsewhere in the file for mesh bed leveling.
"Plate" is used mainly to talk about the sheets, so I think Sengeplanering is more correct.
Should be consistent at least.

OS-kar replied:
okay lets stick to "seng" then to be consistent.

3d-gussner replied:
Thanks guys, what is your verdict now. Both agree that it stays like I "translated"?
BTW: I looked up the PO file for similar text and reused it to be consistent.

arekm commented on lang/po/Firmware_pl.po:
```diff
@@ -2568,6 +2568,11 @@ msgstr "Załaduj nowy filament lub wyładuj poprzedni."
+#. MSG_MBL_FAILED_Z_CAL c=20 r=4
+#: ../../Firmware/Marlin_main.cpp:2976
+msgid "Mesh bed leveling failed. Please run Z calibration."
+msgstr "Poziomowanie stolu. Proszę uruchomić kalibrację Z."
```
Is 63 characters ok and is "ł" available? If yes/yes then:

"Poziomowanie stołu nieudane. Proszę uruchomić kalibrację Z."

"Poziomowanie stołu" means "bed leveling". "nieudane" means "failed".

3d-gussner replied:
@arekm Thanks for your comment. At this moment we replace all diacritics with aA-zZ chars but we want to keep the po files with these. So "ł" is not shown on LCD (at this moment see https://github.com/prusa3d/Prusa-Firmware/pull/3383) but appreciated to have these in the po files.
Auto correction in PO editors works way better this way.

3d-gussner committed:
Thanks to everyone for the translation updates.
@ingbrzy @leptun @petrubecheru @Hauzman @AttilaSVK @sarusani @Painkiller56 @vintagepc @stelgenhof
@wavexx @OS-kar @trondkla @arekm

I had to remove the French translation as it exceeds the limt. Only 53 bytes are free.

Prime1910 commented:
Croatian is good
